packages/contracts/src/contracts/current/protocol/Exchange/mixins/MExchangeCore.sol

```diff
@@ -16,21 +16,22 @@
 
 */
 
-pragma solidity ^0.4.19;
+pragma solidity ^0.4.21;
+pragma experimental ABIEncoderV2;
 
-contract MExchangeCore {
+import "../LibOrder.sol";
+
+contract MExchangeCore is LibOrder {
     
     function fillOrder(
-          address[5] orderAddresses,
-          uint256[6] orderValues,
-          uint256 takerTokenFillAmount,
-          bytes signature)
-          public
-          returns (uint256 takerTokenFilledAmount);
+        Order order,
+        uint256 takerTokenFillAmount,
+        bytes signature)
+        public
+        returns (uint256 takerTokenFilledAmount);
```

Comment:
Is this the proper amount of indentation?

Reply:
Yep